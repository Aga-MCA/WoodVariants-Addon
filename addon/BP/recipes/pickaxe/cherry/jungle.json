{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "cherry_jungle_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:jungle_stick"
      },
      "P": {
        "item": "aga_wood_variants:cherry_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:cherry_jungle_pickaxe"
    }
  }
}