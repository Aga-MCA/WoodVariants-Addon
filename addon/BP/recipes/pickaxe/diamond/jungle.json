{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "diamond_jungle_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:jungle_stick"
      },
      "P": {
        "item": "minecraft:diamond"
      }
    },
    "result": {
      "item": "aga_wood_variants:diamond_jungle_pickaxe"
    }
  }
}