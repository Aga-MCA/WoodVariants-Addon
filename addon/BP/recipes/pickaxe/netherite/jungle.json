{
  "format_version": "1.17",
  "minecraft:recipe_smithing_transform": {
    "description": {
      "identifier": "netherite_jungle_pickaxe"
    },
    "tags": [
      "smithing_table"
    ],
    "template": "minecraft:netherite_upgrade_smithing_template",
    "base": "aga_wood_variants:diamond_jungle_pickaxe",
    "addition": "minecraft:netherite_ingot",
    "result": "aga_wood_variants:netherite_jungle_pickaxe"
  }
}