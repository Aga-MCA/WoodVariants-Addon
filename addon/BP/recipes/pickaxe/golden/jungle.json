{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_jungle_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:jungle_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_jungle_pickaxe"
    }
  }
}